site/src/components/Tooltips/HelpTooltip/HelpTooltip.stories.tsx

```diff
@@ -15,10 +15,10 @@ export default {
 
 const Template: Story<HelpTooltipProps> = (args) => (
   <HelpTooltip {...args}>
-    <HelpTooltipTitle>What is template?</HelpTooltipTitle>
+    <HelpTooltipTitle>What is a template?</HelpTooltipTitle>
     <HelpTooltipText>
-      With templates you can create a common configuration for your workspaces using Terraform. So, you and your team
-      can use the same environment to deliver great software.
+      With templates, you can create a common configuration for your team`&apos;`s workspaces using Terraform so that
```

Comment:
```suggestion
      A template is a common configuration for your team`&apos;`s workspaces.
```

The title is "What is a template?" so beginning with "A template is" is more expected response. The ending of "can use the same the same environment to deliver great software" is marketing speak that doesn't exist in our other tooltips, and shouldn't exist in the product.